WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -428,6 +428,18 @@ public void showPostActivity(long siteId, long postId) {
         ReaderActivityLauncher.showReaderPostDetail(this, siteId, postId);
     }
 
+     public void showScanActivityForSite(long siteId) {
+         SiteModel site = mSiteStore.getSiteBySiteId(siteId);
+         if (site == null) {
```

Comment:
Suggestion (💡): What if the duplicate with the toast and comments is reduced like that below:
```
     SiteModel site = getSiteOrToast(siteId);
     if (site != null) {
         ActivityLauncher.viewScan(this, site);
     }

    @Nullable
    private SiteModel getSiteOrToast(long siteId) {
        SiteModel site = mSiteStore.getSiteBySiteId(siteId);
        if (site == null) {
            // One way the site can be null: new site created, receive a notification from this site,
            // but the site list is not yet updated in the app.
            ToastUtils.showToast(this, R.string.blog_not_found);
        }
        return site;
    }
```

Reply:
Good call ;)! Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13941/commits/16f0af7d80dc9dece4aecc3f29ad6de43161f692

Reply:
🙏